traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Question.java

```diff
@@ -16,13 +21,15 @@
     private String description;
     private List<ValidationRule> validationRules;
     private QuestionType type;
+    private Object suggestedAnswer;
 
-    Question(@NonNull String id, @NonNull String title, String description, @NonNull QuestionType type, @NonNull List<ValidationRule> rules) {
+    Question(@NonNull String id, @NonNull String title, String description, @NonNull QuestionType type, @NonNull List<ValidationRule> rules, Object suggestedAnswer) {
```

Comment:
Object? This should be a more specific type

Reply:
Create an abstract SuggestedAnswer type or reuse Answer type

and use implementing classes for safety

Reply:
One thing to point out here however, you are including the suggested answer as part of the question. it would be best if it is in the BookingForm type. 

All this logic should be internal to the SDK, and when the SDK gives out a BookingForm, the Answers are already there. That way no more UI work is required either

Reply:
i can not change this. this is what we get from server side. ios has an "any" type like kotlin, but java's closest type is Object. if you look at purchase form i used Answer type and everything is being handled in sdk. i can probably make Question generic but i think thats another can of worm.

this is simply the identical implementation to the IOS

Reply:
@ata-n i changed it and i made a SuggestedAnswer type (using same questionType was messy. its smelly to store value in QuestionType). its all good now. except that i could have re-used Answer type if it didnt want the whole question object in constructor and it only needed question id (it uses it to make sure the question id belongs to a question with correct type).